docs/RichTextInput.md

```diff
@@ -172,9 +171,9 @@ const MyRichTextInput = ({ size, ...props }) => (
 );
 ```
 
-## `ref`
+## Reference the editor
 
-You might need access to the editor to use [the dedicated API](https://tiptap.dev/docs/editor/api/editor). To do it, you can pass a `ref` to your `<RichTextInput>` component and use it:
+You might need access to the editor to use [the dedicated API](https://tiptap.dev/docs/editor/api/editor). To do it, you can pass a `ref` to the `onCreate` function in the `editorOptions` prop of your `<RichTextInput>` component:
```

Comment:
This doesn't look right. You don't pass a ref to the function. 
```suggestion
You might need access to the editor to use [the dedicated API](https://tiptap.dev/docs/editor/api/editor). To do it, use the `onCreate` function in the `editorOptions` prop of your `<RichTextInput>` component which returns the editor:
```

Reply:
I didn't change `you can pass a 'href'` to `use` because there was a conflit with [this comment](#discussion_r1530631189)

Reply:
But `which returns the editor` added